plugins/ftdidmx/FtdiDmxThread.cpp

```diff
@@ -73,15 +99,164 @@ bool FtdiDmxThread::WriteDMX(const DmxBuffer &buffer) {
   }
 }
 
+void FtdiDmxThread::SendRDMRequest(ola::rdm::RDMRequest *request,
+                                   ola::rdm::RDMCallback *callback) {
+  ola::thread::MutexLocker locker(&m_rdm_mutex);
+  if (m_pending_request == nullptr) {
+      m_pending_request = request;
+      m_rdm_callback = callback;
+  } else {
+    OLA_WARN << "Unable to queue RDM request, RDM operation already pending";
+  }
+}
+
+void FtdiDmxThread::RunFullDiscovery(ola::rdm::RDMDiscoveryCallback *callback) {
+  m_discovery_agent.StartFullDiscovery(
+        ola::NewSingleCallback(this,
```

Comment:
Indent by four, or alternatively lay it out like this:
```
void FtdiDmxThread::RunFullDiscovery(ola::rdm::RDMDiscoveryCallback *callback) {
  m_discovery_agent.StartFullDiscovery(ola::NewSingleCallback(
      this,
      &FtdiDmxThread::DiscoveryComplete,
      callback));
}
```

Reply:
Fixed